resources/gen-version.js

```diff
@@ -4,21 +4,31 @@ const fs = require('fs');
 
 const { version } = require('../package.json');
 
-const versionMatch = /^(\d+)\.(\d+)\.(\d+)-?(.*)?$/.exec(version);
-if (!versionMatch) {
-  throw new Error('Version does not match semver spec: ' + version);
+if (require.main === module) {
```

Comment:
@PabloSzx Please revert, we shouldn't run `npm version` twice and create two sets of tags and duplicate other steps.
It just that `npm run build:npm` should now produce two folder standard and ESM.
And `16.0.1` is `16.0.1` even if it's published as ESM.
If devs for some reason want to know if it's ESM or not they should import `package.json`